### PR TITLE
Add Scotto37 handwired keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Adds the Scotto37 handwired keyboard and default keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scotto37/info.json
Comment (zvecr) [inline keyboards/handwired/scottokeebs/scotto37/info.json:1]: Please rename this file to `keyboard.json`

File: keyboards/handwired/scottokeebs/scotto37/rules.mk
Comment (zvecr) [inline keyboards/handwired/scottokeebs/scotto37/rules.mk:1]: Please remove this file.

File: keyboards/handwired/scottokeebs/scotto37/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define options
+#define TAPPING_TERM 135
+#define PERMISSIVE_HOLD
+#define TAPPING_TERM_PER_KEY
```
Comment (zvecr) [inline keyboards/handwired/scottokeebs/scotto37/keymaps/default/config.h:23]: This config option does provide any value when the corresponding code is not implemented.
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scotto37/rules.mk
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto37/rules.mk:1]: Resolved in [e3104c2](https://github.com/qmk/qmk_firmware/pull/23938/commits/e3104c27161f9aee4be6e28ba4e5fc214d21112c)

File: keyboards/handwired/scottokeebs/scotto37/info.json
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto37/info.json:1]: Resolved in [e3104c2](https://github.com/qmk/qmk_firmware/pull/23938/commits/e3104c27161f9aee4be6e28ba4e5fc214d21112c)

File: keyboards/handwired/scottokeebs/scotto37/keyboard.json
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scotto37/keyboard.json:1]: Please run `qmk format-json -i` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scotto37/keyboard.json
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto37/keyboard.json:1]: Fixed in [72688e4](https://github.com/qmk/qmk_firmware/pull/23938/commits/72688e4720b15a05cc2b48a7932ef39818043e67)

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**